------------------------------------------------------------
revno: 244
committer: T(A)ILS developper <amnesia@boum.org>
branch nick: trunk
timestamp: Sat 2011-01-29 12:11:03 +0100
message:
  Effectively close onboard through the GtkStatusIcon menu
  
  Remove the test as it works well with GtkStatusIcon
diff:
=== modified file 'Onboard/OnboardGtk.py'
--- Onboard/OnboardGtk.py	2010-08-19 10:11:26 +0000
+++ Onboard/OnboardGtk.py	2011-01-29 11:11:03 +0000
@@ -87,8 +87,7 @@
 
         # create status icon
         self.status_icon = Indicator(self._window)
-        if self.status_icon.is_appindicator():
-            self.status_icon.connect("quit-onboard", self.do_quit_onboard)
+        self.status_icon.connect("quit-onboard", self.do_quit_onboard)
 
         # Callbacks to use when icp or status icon is toggled
         config.show_status_icon_notify_add(self.show_hide_status_icon)
